projects/netgrif-components-core/src/lib/impersonation/services/impersonation-user-select.service.ts

```diff
@@ -13,37 +10,50 @@ import {SimpleFilter} from '../../filter/models/simple-filter';
 import {UserImpersonationConstants} from '../models/user-impersonation-constants';
 import moment from 'moment';
 import {UserService} from '../../user/services/user.service';
+import {MatDialog} from '@angular/material/dialog';
+import {
+    NAE_ADMIN_IMPERSONATE_DIALOG_COMPONENT,
+    NAE_USER_IMPERSONATE_DIALOG_COMPONENT
+} from '../../dialog/injection-tokens';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ImpersonationUserSelectService {
 
     constructor(protected _log: LoggerService,
-                protected _sideMenuService: SideMenuService,
                 protected _snackBar: SnackBarService,
                 protected _impersonation: ImpersonationService,
                 protected _user: UserService,
-                @Optional() @Inject(NAE_USER_IMPERSONATE_COMPONENT) protected _userImpersonateComponent: any,
-                @Optional() @Inject(NAE_ADMIN_IMPERSONATE_COMPONENT) protected _adminImpersonateComponent: any,
+                protected _dialog: MatDialog,
+                @Optional() @Inject(NAE_USER_IMPERSONATE_DIALOG_COMPONENT) protected _userImpersonateComponent: any,
+                @Optional() @Inject(NAE_ADMIN_IMPERSONATE_DIALOG_COMPONENT) protected _adminImpersonateComponent: any,
     ) {}
 
     /**
      * Opens side menu with users who can be impersonated by logged user
      */
     public selectImpersonate(): void {
         if (this.isAdmin()) {
-            this._sideMenuService.open(this._adminImpersonateComponent, SideMenuSize.LARGE, this.injectedData()).onClose.subscribe(event => {
+            const dialogRef = this._dialog.open(this._adminImpersonateComponent, {
+                panelClass: "dialog-responsive",
```

Comment:
I noticed that "dialog-responsive" is repeating several times, can it be a string constant?